templates/content-page.php

```diff
@@ -1,13 +1,11 @@
 <?php while (have_posts()) : the_post() ?>
-  <article class="page-content clearfix">
-    <div class="visible-print">
+  <article>
+    <div class="govuk-visible-print">
```

Comment:
I think the intention of the original `visible-print` class was that the content would be visible when printed, but not on the screen. The `govuk-visible-print` class doesn't exist, so this change results in the permalink being displayed on the screen.